Classes/Service/DestinationDataImportService.php

```diff
@@ -475,7 +476,11 @@ private function setTexts(array $texts): void
             if ($text['rel'] == 'teaser' && $text['type'] == 'text/plain') {
                 $this->tmpCurrentEvent->setTeaser(str_replace("\n\n", "\n", $text['value']));
             }
-            if ($text['rel'] == 'PRICE_INFO' && $text['type'] == 'text/plain') {
+            if ($shouldSetPrice && $text['rel'] == 'PRICE_INFO' && $text['type'] == 'text/plain') {
+                $this->tmpCurrentEvent->setPriceInfo(str_replace("\n\n", "\n", $text['value']));
+            }
+            if ($shouldSetPrice && $text['rel'] == 'PRICE_INFO_EXTRA' && $text['type'] == 'text/plain') {
+                $shouldSetPrice = false;
                 $this->tmpCurrentEvent->setPriceInfo(str_replace("\n\n", "\n", $text['value']));
             }
```

Comment:
```suggestion
            if ($shouldSetPrice && $text['rel'] == 'PRICE_INFO_EXTRA' && $text['type'] == 'text/plain') {
                $shouldSetPrice = false;
                $this->tmpCurrentEvent->setPriceInfo(str_replace("\n\n", "\n", $text['value']));
            }
            if ($shouldSetPrice && $text['rel'] == 'PRICE_INFO' && $text['type'] == 'text/plain') {
                $this->tmpCurrentEvent->setPriceInfo(str_replace("\n\n", "\n", $text['value']));
            }
```
```suggestion
            if ($shouldSetPrice && $text['rel'] == 'PRICE_INFO' && $text['type'] == 'text/plain') {
                $this->tmpCurrentEvent->setPriceInfo(str_replace("\n\n", "\n", $text['value']));
            }
            if ($shouldSetPrice && $text['rel'] == 'PRICE_INFO_EXTRA' && $text['type'] == 'text/plain') {
                $shouldSetPrice = false;
                $this->tmpCurrentEvent->setPriceInfo(str_replace("\n\n", "\n", $text['value']));
            }
```

This can avoid calling the setPriceInfo method twice.